.gitignore

```diff
@@ -49,6 +49,7 @@ playwright-report
 # build outputs
 lib
 libDev
+libESM
```

Comment:
Please check also other "ignore" files in monorepo root, there is multiple of them

Reply:
Right, updated https://github.com/trezor/trezor-suite/pull/16470/commits/ee0fb8c16334776deb3b1d3a05d88b040d62ecbc